README.md

```diff
@@ -22,37 +23,46 @@ In addition to deploying new clusters, this playbook also support the deployment
 
 
 ## Index
```

Comment:
It seems something went wrong here :) Do we really need to update the index?

Reply:
Oops, my mistake. It's the automatic backup on VSCode that automatically reformats the code, it wasn't intentional. Thankfully, you have a keen eye. I'll fix it right away.

Reply:
Fixed